lib/datadog/tracing/configuration/settings.rb

```diff
@@ -173,6 +173,24 @@ def self.extended(base)
                 o.lazy
               end
 
+              # Enable 128 bit trace id generation.
+              #
+              # @default `DD_TRACE_128_BIT_TRACEID_GENERATION_ENABLED` environment variable, otherwise `false`
+              # @return [Boolean]
+              option :trace_id_128_bit_generation_enabled do |o|
+                o.default { env_to_bool(Tracing::Configuration::Ext::ENV_TRACE_ID_128_BIT_GENERATION_ENABLED, false) }
+                o.lazy
+              end
+
+              # Enable 128 bit trace id injected for logging.
```

Comment:
Let's add a comment here saying it's not supported by the backend yet. Also, we shouldn't surface it in our `GettingStarted.md` yet.